index.js

```diff
@@ -28,7 +28,14 @@ const exampleMovies = require("./movies");
       "James and the Giant Peach",
     ];
  */
-function getAllMovieTitles() {}
+function getAllMovieTitles(movies) {
+  let titalArr = [];  
```

Comment:
Watch the spelling here... I'd probably call this something like `movieTitleArray`